docs/nf4_science/imaging/03_inputs.md

```diff
@@ -0,0 +1,210 @@
+# Part 3: Organizing inputs
+
+In Part 2, we ran molkart with multiple parameters on the command line.
+Now we'll learn two better approaches for managing inputs: **parameter files** and **samplesheets**.
+
+## 1. Using parameter files
+
+### 1.1. The problem with long command lines
+
+Recall our command from Part 2:
+
+```bash
+nextflow run ./molkart \
+  --input 'data/samplesheet.csv' \
+  --mindagap_tilesize 90 \
+  --mindagap_boxsize 7 \
+  --mindagap_loopnum 100 \
+  --clahe_pyramid_tile 368 \
+  --segmentation_method "cellpose" \
+  --outdir results
+```
+
+This works, but it's hard to reproduce, share, or modify.
+What if you need to run the same analysis again next month?
+What if a collaborator wants to use your exact settings?
+
+### 1.2. Solution: Use a parameter file
+
+Create a file called `params.yaml`:
+
+```yaml title="params.yaml"
+input: "data/samplesheet.csv"
+outdir: "results"
+mindagap_tilesize: 90
+mindagap_boxsize: 7
+mindagap_loopnum: 100
+clahe_pyramid_tile: 368
+segmentation_method: "cellpose"
+```
+
+Now your command becomes:
+
+```bash
+nextflow run ./molkart -params-file params.yaml
+```
+
+That's it! The parameter file documents your exact configuration and makes it easy to rerun or share.
+
+### 1.3. Overriding parameters
+
+You can still override specific parameters from the command line:
+
+```bash
+nextflow run ./molkart -params-file params.yaml --outdir test_results
+```
+
+This is useful for quick tests without editing the file.
```

Comment:
```suggestion
The above line changes the `--outdir` name to be `test_results` instead of the `results` name as specified in the `params.yaml` file. This is useful for quick tests without editing the params file.
```